resip/stack/GenericPidfContents.hxx

```diff
@@ -71,6 +71,7 @@ public:
       EncodeStream& encode(EncodeStream& str, Data indent) const;
    };
    const NodeList& getRootNodes() const { checkParsed(); return mRootNodes; }
+   void setRootNodes(const NodeList& nodeList);
```

Comment:
in this case, we'll accept it as part of this pull request but in future I would encourage you to submit changes like that atomically.